modular_citadel/code/modules/mob/cit_emotes.dm

```diff
@@ -77,8 +77,7 @@
 	restraint_check = TRUE
 	emote_type = EMOTE_AUDIBLE
 	sound = 'sound/effects/snap01.ogg'
-	mob_type_allowed_typecache = list(/mob/living/carbon/)
-
+	mob_type_allowed_typecache = list(/mob/living/carbon, /mob/living/silicon/pai)
```

Comment:
Now, once you have replaced all those typechecks with proper parent return value checks, copy this line to each following typepath too, thank you.